### PR TITLE
(#872) Step Indicator Component

### LINKED ISSUE #872: Step Indicator USWDS Pass Through to NCIDS
### USWDS Component Name
[Step Indicator](https://designsystem.digital.gov/components/step-indicator/)

### USWDS Component Packages
* Package usage: `@forward "usa-step-indicator";`
* Dependencies: `uswds-fonts`, `uswds-helpers`

### Subtasks
- [x] Make a folder with the component name under uswds-native
- [x] Copy USWDS twig and any JSON
- [x] Make an index.scss for the stories
- [x] Copy and hack up USWDS stories.js file to use our story format
- [x] Create scenarios.js by adding a scenario for each story in stories.js
- [x] Add link to the USWDS documentation on the NCIDS documentation site
- [x] **Patch font size to xs**

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #872

### CODE REVIEW

File: packages/ncids-css/package.json [hunk 1]
```diff
@@ -119,7 +118,7 @@
 		]
 	},
 	"uswdsExcludeComments": {
-		"//": "usa-content, usa-dark-background, usa-display, usa-intro, usa-link, usa-paragraph, usa-prose all required for uswds-typography.",
+		"//": "usa-content, usa-dark-background, usa-display, usa-intro, usa-link, usa-paragraph, usa-step-indicator, usa-prose all required for uswds-typography.",
```
Comment (olitharp-nci) [inline packages/ncids-css/package.json:121]: change this back too - step indicator shouldn't be required for uswds-typography to work

File: testing/ncids-css-testing/stories/uswds-native/usa-step-indicator/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@use "styles/ncids";
+@forward "usa-step-indicator";
+@forward "uswds-helpers";
```
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/uswds-native/usa-step-indicator/src/index.scss:3]: add end of line to end of file so github happy

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

File: testing/ncids-css-testing/stories/uswds-native/usa-step-indicator/src/index.scss (hunk 2 above)
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/uswds-native/usa-step-indicator/src/index.scss:3]: oh my bad. uswds-global also imports uswds-helpers. sorry, should have caught that at my last look over

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Review by olitharp-nci — APPROVED** [review]

**Review by dlescarbeau — APPROVED** [review]

**Review by monika-jaeger — APPROVED** [review]
Approved for product.

**Pull request merged**